README.md

```diff
@@ -25,7 +25,37 @@ Read
 [_PDD in Action_](http://www.yegor256.com/2017/04/05/pdd-in-action.html)
 and watch [this webinar](https://www.youtube.com/watch?v=nsYGC2aUwfQ).
 
-You should have [Ruby installed](https://www.ruby-lang.org/en/documentation/installation/).
+## Installation
+<details><summary>
+
+### Prerequisites: `Ruby installed` and `Libmagic`
+</summary>
+<details><summary>
+
+#### Ruby version 2.7+
+</summary>
+[Ruby installation instruction](https://www.ruby-lang.org/en/documentation/installation/)
```

Comment:
I see that at https://github.com/pnatashap/pdd/tree/227 this link wasn't rendered properly.

Reply:
fixed